src/puya/awst/nodes.py

```diff
@@ -1454,6 +1743,10 @@ def accept(self, visitor: StatementVisitor[T]) -> T:
 
 @attrs.frozen
 class Emit(Expression):
+    """
+    Emits an ARC28 log event using the provided signature and struct
```

Comment:
Hmmm, this node could probably use a bit of refinement - the signature should probably either be derived from value, or the value should be a sequence/mapping. But that's for a seperate issue. 

```suggestion
    Emits an ARC-28 log event using the provided signature, and the serialisation of the value.
```